javascript/elements/updates_for_element.js

```diff
@@ -73,6 +73,35 @@ export default class UpdatesForElement extends SubscribingElement {
       }
 
       template.innerHTML = String(html[url(blocks[i])]).trim()
+
+      const reloadingTurboFrames = [
+        ...template.content.querySelectorAll(
+          'turbo-frame[src]:not([loading="lazy"])'
+        )
+      ]
+
+      await Promise.all(
+        reloadingTurboFrames.map(frame => {
+          return new Promise(async resolve => {
+            const frameResponse = await fetch(frame.getAttribute('src'), {
+              headers: { 'Turbo-Frame': frame.id }
```

Comment:
Tangent: what's the deal with `X-Cable-Ready` on line 66? It doesn't occur anywhere else in the CableReady codebase.

Reply:
cf https://github.com/stimulusreflex/cable_ready/pull/159

To do something like
`before_action { layout false if request.headers['X-Cable-Ready'] == 'update' }` etc etc